jolt-core/src/poly/commitment/commitment_scheme.rs

```diff
@@ -23,9 +24,14 @@ pub trait CommitmentScheme<ProofTranscript: Transcript>: Clone + Sync + Send + '
     type Proof: Sync + Send + CanonicalSerialize + CanonicalDeserialize;
     type BatchedProof: Sync + Send + CanonicalSerialize + CanonicalDeserialize;
 
-    fn setup(max_len: usize) -> Self::Setup;
-    fn commit(poly: &MultilinearPolynomial<Self::Field>, setup: &Self::Setup) -> Self::Commitment;
-    fn batch_commit<U>(polys: &[U], gens: &Self::Setup) -> Vec<Self::Commitment>
+    fn setup_prover(max_len: usize) -> Self::ProverSetup;
+    fn setup_verifier(setup: &Self::ProverSetup) -> Self::VerifierSetup;
+    fn srs_size(setup: &Self::ProverSetup) -> usize;
```

Comment:
do we need this function? imo the "size" of an SRS is kind of ambiguous, so I'd prefer to delete it if it's unneeded

Reply:
ah right –– let's leave it then